src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java

```diff
@@ -39,4 +38,19 @@ String handleRatingsBadValueException(HttpServletRequest request, Throwable ex)
         return "Rating can't be less than 0 and more than 5";
     }
 
+    @ExceptionHandler(ImageRepositorySizeQuotaExceededException.class)
+    ResponseEntity<MessageDTO> handleImageRepositorySizeQuotaExceededException() {
+        return new ResponseEntity<>(new MessageDTO("You have exceeded your quota for uploading images. You should delete some images before new upload."), HttpStatus.CONFLICT);
+    }
+
+    @ExceptionHandler(CanNotBeDeletedException.class)
+    ResponseEntity<MessageDTO> handleCanNotBeDeletedException(){
+        return new ResponseEntity<MessageDTO>(new MessageDTO("Current image is already in use!"), HttpStatus.CONFLICT);
+    }
+
+    @ExceptionHandler(NotOwnerOperationException.class)
+    ResponseEntity<MessageDTO> handleNotOwnerOperationException(){
+        return new ResponseEntity<MessageDTO>(new MessageDTO("Operation is not allowed for current user!"), HttpStatus.CONFLICT);
+    }
+
```

Comment:
It drops the exception with message to UI, when user is trying to delete image that is not his own